lib/exabgp/util/__init__.py

```diff
@@ -58,10 +58,16 @@ def padding(n):
 def concat_strs(*items):
 	return ''.join(items)
 
-
 def concat_bytes(*items):
 	return b''.join(items)
 
+# same with iterators/lists
+
+def concat_strs_i(iterable):
```

Comment:
I would prefer merging as is, if only to unbreak master.

Then we can consider merging both under a same function, based on testing if there is only one argument which can be iterated over... is this what you have in mind ?